Increased battery life: added extra sections on tweaking Grub bootloader and downagrading to Unity 2D

myrtleTree33 opened the pull request:
Increased battery life: added extra sections on tweaking Grub bootloader and downagrading to Unity 2D

Commit:
added extra sections on tweaking Grub bootloader and fdownagrading to
unity2D for extra battery performance

longsleep commented:
Thanks for sharing this. I suggest to remove the Unity 2D suggstion as a seperate section as it is deprecated and should not really be used. The boot parameters section is great.

Commit:
cleared explanation and added link to Grub bootloader tweak / power consumption
affecting Intel notebooks

myrtleTree33 commented:
@longsleep Great, yes it is indeed deprecated.  Did some research, seems that 3D rendering does not affect much battery life on integrated Intel graphics chips.

Added explanation to clarify fix, and removed the Unity2D section.

longsleep commented:
Merged. Thanks!
